config/application.rb

```diff
@@ -4,6 +4,7 @@
 
 require "rails/all"
 require_relative "../lib/jwt_middleware"
+require "ostruct"
 
```

Comment:
lol

Reply:
Ok you got me - that's probably a symptom of something else but it fixed a bunch of these errors:

```
NameError:
  uninitialized constant GraphQL::Compatibility::ExecutionSpecification::SpecificationSchema::OpenStruct
```

Reply:
Im laughing at ruby/removing this from the core library - its everywhere